client/test/periodic.test.js

```diff
@@ -13,6 +13,13 @@ describe('The periodic module', () => {
     });
   });
 
+  describe('runDaily()', () => {
+    it('allows registration of a daily callback', () => {
+      let p = periodic(app);
+      assert.ok(p.runDaily('jest-fun', () => { }));
```

Comment:
IIUC this does not really test the daily run, just that calling the method does not fail.